### ISSUE TITLE
Accessibility - Contrast - Main Menu

### ISSUE THREAD

**Issue by adam3smith** [QualitativeDataRepository/dataverse#83]
![image](https://user-images.githubusercontent.com/128986/133820947-989cb9d7-6e43-4633-8f89-377cbe77f8f6.png)

The menu headers have insufficient contrast because of their opacity. Let's move that up to 0.75

**Comment by qqmyers** [comment]
fixed on dev, though Axe still reports 4 issues when the menu is open - looks like it assumes the text under the popup has low contrast since it is covered by the background of the popup.
